akka-actor/src/main/mima-filters/2.5.18.backwards.excludes

```diff
@@ -65,7 +65,7 @@ ProblemFilters.exclude[DirectMissingMethodProblem]("akka.pattern.BackoffOptionsI
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.pattern.BackoffOptionsImpl.this")
 ProblemFilters.exclude[MissingTypesProblem]("akka.pattern.BackoffOptionsImpl$")
 # private
-ProblemFilters.exclude[DirectMissingMethodProblem]("akka.pattern.BackoffOnRestartSupervisor.this")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.pattern.internal.BackoffOnRestartSupervisor.this")
```

Comment:
These old excludes should not be changed, the 2.5.21 one should contain all filtering for this change.

Reply:
Fixed in 35207d5